src/components/entry/EntryDetails.js

```diff
@@ -159,12 +152,14 @@ const EntryDetails = ({ isDesktop }) => {
         <Description>
           <p>{locationData.description}</p>
           <small>Last Updated {formatISOString(locationData.updated_at)}</small>
-          <Button>
-            <Star /> Review
-          </Button>
-          <Button secondary>
-            <Flag /> Report
-          </Button>
+          <div>
```

Comment:
nit: `<>` (![short fragment syntax](https://reactjs.org/docs/fragments.html#short-syntax)) cooler and more succinct than `<div>` if you don't need css classes defined

Reply:
Thanks Neeraj. We need a div here for the margin to apply to the buttons as a whole, and not each of them.